src/main/webapp/util/FundingsSourceReportConstants.js

```diff
@@ -0,0 +1,23 @@
+export const SORT_NAME_INDEX = {
+  'type': 0,
+  'projectKey': 1,
+  'summary': 2,
+  'status': 3,
+  'protocol': 4, // Not implemented
+  'pis': 5, // Not implemented
+  'source': 6,
+  'name': 7,
+  'awardNumber': 8
+};
+
+export const STYLES = {
+  issueTypeWidth: '96px',
+  projectKeyWidth: '109px',
+  titleWidth: '280px',
+  statusWidth: '98px',
+  pisWidth: '80px',
+  protocolWidth: '94px',
+  fundingNameWidth: '124px',
+  generalWidth: '136px'
+};
```

Comment:
Note that I strongly prefer this kind of inline style declaration over external css. This PR does mix the two concepts though (see `TableComponent.css`).  It would be nice to keep moving in this direction.